ui/apps/platform/cypress/integration/policies/saveAsCustomResource.test.ts

```diff
@@ -0,0 +1,143 @@
+import { selectors } from '../../constants/PoliciesPage';
+import { hasFeatureFlag } from '../../helpers/features';
+import withAuth from '../../helpers/basicAuth';
+import {
+    doPolicyPageAction,
+    doPolicyRowAction,
+    visitPolicies,
+    visitPolicy,
+} from '../../helpers/policies';
+import { interceptAndWatchRequests } from '../../helpers/request';
+
+const saveAsUrl = '/v1/policies/save-as';
+
+const routeMatcherMapForPolicySaveAs = {
+    [saveAsUrl]: { method: 'POST', url: saveAsUrl },
+};
+
+const firstTableRowSelector = 'tbody:nth-child(2) tr:nth-child(1)';
```

Comment:
Forgive if this is a repeat comment. Or maybe Internet went from slow to no when I tried to add it.

Is `nth-child(2)` copy-paste? From the context, I expected `(1)` instead.

Reply:
This is in fact a copy-paste from the _export_ policy test. This works since the first `tbody` is the second child of `table`, with `thead` being the first. I suppose `tbody:nth-of-type(1)` could be more accurate if we want to make the change.